dla_cnn/desi/DesiMock.py

```diff
@@ -119,8 +118,7 @@ def get_data(start_point = 0, end_point = self.data_size):
         else:
             get_data(start_point=self.split_point_rz)
 
-        if rebin:
-            _rebin(sightline, best_v[camera])
+
```

Comment:
I think you meant to keep `_rebin` and remove `_normalize`

Reply:
Sure, I have noticed and corrected it.